README.md

```diff
@@ -45,135 +45,51 @@ This environment variable will be used in the deployment steps.
 
+<!--TODO Change to stable alfresco-infrastructure that inculdes alfresco-identity-service AUTH-193-->
 ```bash
-helm repo update
 
-cat <<EOF > ingressvalues.yaml
-controller:
-  config:
-    ssl-redirect: "false"
-  scope:
-    enabled: true
-EOF
-
-helm install stable/nginx-ingress --version=0.14.0 -f ingressvalues.yaml \
-  --namespace $DESIREDNAMESPACE
-```
-
-### Optional
-
-If you want your own certificate set here you should create a secret from your cert files:
-
-```bash
-kubectl create secret tls certsecret --key /tmp/tls.key --cert /tmp/tls.crt \
-  --namespace $DESIREDNAMESPACE
-
-#Then deploy the ingress with following settings
-
-cat <<EOF > ingressvalues.yaml
-controller:
-  config:
-    ssl-redirect: "false"
-  scope:
-    enabled: true
-  publishService:
-    enabled: true
-  extraArgs:
-    default-ssl-certificate: $DESIREDNAMESPACE/certsecret
-EOF
-
-helm install stable/nginx-ingress --version=0.14.0 -f ingressvalues.yaml \
-  --namespace $DESIREDNAMESPACE
-```
-
-If you
-
-* created the cluster in AWS using [kops](https://github.com/kubernetes/kops/)
-* have a matching SSL/TLS certificate stored in [AWS Certificate Manager](https://aws.amazon.com/certificate-manager/)
-* are using a zone in [Amazon Route 53](https://aws.amazon.com/route53/)
-
-Kubernetes' [External DNS](https://github.com/kubernetes-incubator/external-dns)
-can autogenerate a DNS entry for you (a CNAME of the generated ELB) and apply
-the SSL/TLS certificate to the ELB.
-
-_Note: AWS Certificate Manager ARNs are of the form `arn:aws:acm:REGION:ACCOUNT:certificate/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`._
+helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+helm repo add alfresco-stable https://kubernetes-charts.alfresco.com/stable
 
-Set `DOMAIN` to the DNS Zone you used when [creating the cluster](https://github.com/kubernetes/kops/blob/master/docs/aws.md#scenario-1b-a-subdomain-under-a-domain-purchasedhosted-via-aws).
 
-```bash
-ELB_CNAME="${DESIREDNAMESPACE}.${DOMAIN}"
-ELB_CERTIFICATE_ARN=$(aws acm list-certificates | \
-  jq '.CertificateSummaryList[] | select (.DomainName == "'${DOMAIN}'") | .CertificateArn')
-
-cat <<EOF > ingressvalues.yaml
-controller:
-  config:
-    ssl-redirect: "false"
-  scope:
-    enabled: true
-  publishService:
-    enabled: true
-  service:
-    targetPorts:
-      http: http
-      https: http
-    annotations:
-      external-dns.alpha.kubernetes.io/hostname: ${ELB_CNAME}
-      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
-      service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '3600'
-      service.beta.kubernetes.io/aws-load-balancer-ssl-cert: ${ELB_CERTIFICATE_ARN}
-      service.beta.kubernetes.io/aws-load-balancer-ssl-ports: https
-EOF
-
-helm install stable/nginx-ingress --version=0.14.0 -f ingressvalues.yaml \
+helm install alfresco-incubator/alfresco-infrastructure --version 3.0.0-SNAPSHOT \
```

Comment:
I added this specific version in order to deploy the infrastructure that actually contains the identity service. This will be removed as part of the AUTH-193, where a stable version of infrastructure will contain the identity service too.

Reply:
Can we add a TODO in the helm chart that state this?